### ISSUE TITLE
Managing Weave Scope deployment descriptors

### ISSUE THREAD

**Issue by sadovnikov** [weaveworks/scope#1507]
I'm from https://github.com/ContainerSolutions/minimesos/ project - running Apache Mesos cluster on a single machine for prototyping and unit testing of Mesos frameworks. 

In `minimesos` default cluster configuration file includes URL to Marathon JSON file for installing Weave Scope. Currently the file is hosted within `minimesos` repository itself https://github.com/ContainerSolutions/minimesos/blob/master/opt/apps/weave-scope.json and URL in cluster configuration file looks like https://raw.githubusercontent.com/ContainerSolutions/minimesos/e2a43362f4581122762c80d8780d09b567783f1a/apps/weave-scope.json

Would you be possible to host this file in Weave Scope repository, so its versioning becomes more transparent. For example, https://raw.githubusercontent.com/weaveworks/scope/v0.13.1/some-path/marathon.json

**Comment by pidster** [comment]
cc @errordeveloper - for consideration re: adding this to the Kubernetes generator.

**Comment by sadovnikov** [comment]
@pidster, @errordeveloper possibly "Kubernetes generator" is wider than its name tell, however Apaches Mesos with Marathon are not directly related to Kubernetes

**Comment by tomwilkie** [comment]
Yes the kubernetes generator could be made to also generate marathon config; but in the mean time getting the config into this repo seems sensible.  @sadovnikov could you make a PR?

**Comment by sadovnikov** [comment]
yes, sure. Let me study this repository structure to pick the most suitable location
